wa_examples/text_wa.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
similar to `html_wa.json`, just returning newline delimited text

Reply:
Makes sense.